Test/Controller/DataControllerTest.java

```diff
@@ -31,29 +31,39 @@
  */
 public class DataControllerTest {
 
-    @Before
-    public void setUp() throws Exception
-    {
-    }
-
-    @Test
-    public void existingSettingsFile() throws Exception
-    {
-    }
-
-    @Test
-    public void getNodes() throws Exception
-    {
-    }
-
-    @Test
-    public void validNodeList() throws Exception
-    {
-    }
-
-    @Test
-    public void loadHubFile() throws Exception
-    {
-    }
+	/** set up method.
```

Comment:
This Javadoc is not useful.  This method, like others, is empty.  While we certainly want to correct the style problems and make sure that everything is properly documented, adding documentation that does not add value is not a good strategy.  Additionally, the formatting of this JavaDoc is not consistent with the JavaDoc we have elsewhere.  A better JavaDoc comment would be:
```
/**
 * Run prior to each unit test to set up the data structures and environment needed by the test.
 *
 * @throws Exception if there are any problems with the test setup, indicating that there is a problem to investigate
 */
```

Also note that JUnit has annotations for methods that run prior each individual unit test method and for methods that run only once prior to every test method in a given test class.  The same is true for the `@After` and related annotations.  You will need to make sure that your documentation accurately reflects whether the method will run before/after each test or before/after all of the tests.

As far as formatting, you of course need to make sure that the comment begins on the correct line (yours is one line too high) and that proper sentence capitalization is used.

Everything I have pointed out here applies for just about every JavaDoc.